### PR TITLE
SEC-168 - There are various kinds of index-linked bonds, not all of which are variable principal

### PR DESCRIPTION
**Pull request by ElisaKendall**
Signed-off-by: Elisa Kendall <ekendall@thematix.com>

## Description

Broaden the definition of fixed income security to note that what is fixed is the payment schedule, not the payment amount, and revised the hierarchy and definitions related to index-linked bonds and inflation-linked bonds in particular

Fixes: #1611 / SEC-168

## Checklist:

- [x] I'm familiar with the [FIBO developer quide](https://github.com/edmcouncil/fibo/blob/master/CONTRIBUTING.md#contributing-to-the-fibo-code). My contribution meets all the requirements described there.
- [x] My contribution follows the [principles of best practices for FIBO](https://github.com/edmcouncil/fibo/blob/master/ONTOLOGY_GUIDE.md).
- [x] My changes have been reconciled with latest master and no merge conflicts remain.
- [x] This PR is related to exactly one issue. The issue is referenced by using a GitHub keyword such as "fixes", "closes", or "resolves".
- [x] Hygiene tests have been applied by a PR with "(WIP)" in title.
- [x] The issue has been tested locally using a reasoner (for ontology changes).

### CODE REVIEW

**Comment by mereolog** [comment]
@ElisaKendall is this ready for review?
(Note that this still has the WIP status.)

**Comment by ElisaKendall** [comment]
> @ElisaKendall is this ready for review? (Note that this still has the WIP status.)

Now after this latest commit it is.  I'll remove the WIP status so that we can integrate the changes.

**Review by jfgemski — APPROVED** [review]

File: SEC/Debt/Bonds.rdf [hunk 1]
```diff
@@ -1456,6 +1496,15 @@
 		<skos:definition>indicates whether a legal opinion exists for a given municipal bond</skos:definition>
 	</owl:DatatypeProperty>
 	
+	<owl:ObjectProperty rdf:about="&fibo-sec-dbt-bnd;isLinkedToFallback">
+		<rdfs:subPropertyOf rdf:resource="&fibo-fnd-rel-rel;refersTo"/>
+		<rdfs:label>is linked to fallback</rdfs:label>
+		<rdfs:domain rdf:resource="&fibo-fbc-fi-fi;FinancialInstrument"/>
+		<rdfs:range rdf:resource="&fibo-sec-dbt-bnd;GovernmentBond"/>
+		<skos:definition>relates an index-linked instrument to a government bond that may be selected by a calculation agent</skos:definition>
```
Comment (rivettp) [inline SEC/Debt/Bonds.rdf:1504]: neither the definition nor the note say anything about "fallback" which is crucial i,.e. that the rate of the fallback bond is used as a substitute for the inflation index if, for example, it is no longer published.

**Review by rivettp — CHANGES_REQUESTED** [review]
A couple of changes needed I think because it is Release

**Review by jfgemski — DISMISSED** [review]

File: SEC/Debt/Bonds.rdf [hunk 2]
```diff
@@ -1020,21 +1038,41 @@
...
 		<skos:definition>medium term coupon bearing treasury obligation with original maturity ranging from two to ten years</skos:definition>
 	</owl:Class>
 	
-	<owl:Class rdf:about="&fibo-sec-dbt-bnd;TreasurySecurity">
+	<owl:Class rdf:about="&fibo-sec-dbt-bnd;UKGovernmentSecurity">
 		<rdfs:subClassOf rdf:resource="&fibo-sec-dbt-bnd;SovereignDebtInstrument"/>
-		<rdfs:label>treasury security</rdfs:label>
+		<rdfs:label>U.K. Government security</rdfs:label>
+		<skos:definition>debt instrument issued by HM Treasury and listed on the London Stock Exchange</skos:definition>
+		<fibo-fnd-utl-av:abbreviation>GILT</fibo-fnd-utl-av:abbreviation>
```
[the remark this answers is not in the document]
Comment (ElisaKendall) [reply SEC/Debt/Bonds.rdf:1068]: ok - fixed

File: SEC/Debt/Bonds.rdf [hunk 3]
```diff
@@ -1020,21 +1038,41 @@
...
 		<rdfs:label>treasury note</rdfs:label>
 		<skos:definition>medium term coupon bearing treasury obligation with original maturity ranging from two to ten years</skos:definition>
 	</owl:Class>
 	
-	<owl:Class rdf:about="&fibo-sec-dbt-bnd;TreasurySecurity">
+	<owl:Class rdf:about="&fibo-sec-dbt-bnd;UKGovernmentSecurity">
 		<rdfs:subClassOf rdf:resource="&fibo-sec-dbt-bnd;SovereignDebtInstrument"/>
-		<rdfs:label>treasury security</rdfs:label>
+		<rdfs:label>U.K. Government security</rdfs:label>
+		<skos:definition>debt instrument issued by HM Treasury and listed on the London Stock Exchange</skos:definition>
```
[the remark this answers is not in the document]
Comment (ElisaKendall) [reply SEC/Debt/Bonds.rdf:1067]: Ok - I've added an explanatory note and the link you've provided in a seeAlso.

File: SEC/Debt/Bonds.rdf (hunk 1 above)
Comment (ElisaKendall) [reply SEC/Debt/Bonds.rdf:1504]: Added your note to the explanatory note.

**Review by rivettp — APPROVED** [review]
Thanks for updates.

**Review by jfgemski — APPROVED** [review]

**Pull request merged**